libcxx/test/std/ranges/range.adaptors/range.stride.view/iterator/operator.pass.cpp

```diff
@@ -0,0 +1,344 @@
+//===----------------------------------------------------------------------===//
```

Comment:
I think this test could be simplified by asserting using the standard iterator concepts instead of re-building them up. Please let me know if I'm missing something.

Reply:
I believe that this is addressed in a6c41bde85d93720fef6ac7ebf55d482da8aad42. Let me know what you think!